python/tank/util/qt_importer.py

```diff
@@ -251,6 +252,9 @@ def _import_pyside2(self):
 
         modules_dict = {"QtCore": QtCore}
 
+        # Add shiboken2 to the modules dict
+        modules_dict["shiboken"] = shiboken2
```

Comment:
Good catch here by the unit tests. We also need to add the shiboken module to `_import_pyside6` - the reason the unit tests did not catch that one is because it does not have PySide6 set up yet.